docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,107 @@
+Reference the `name` you configure for each individual component in the `merged` sensor's configuration attributes as shown:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your movement sensor, select the `movement-sensor` type, and select the `merged` model.
+
+Click **Create Component**.
+
+{{< imgproc src="/components/movement-sensor/merged-config-builder.png" alt="Creation of an `merged` movement sensor in the Viam app config builder." resize="600x" >}}
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-sensor-name>",
+      "type": "movement_sensor",
+      "model": "merged",
+      "attributes": {
+        "position": ["<your-gps-sensor-name-1>", "<your-gps-sensor-name-2>"],
+        "orientation" : ["<your-imu-sensor-name-1>"],
+        "compass_heading" : ["<your-gps-sensor-name-1>"],
+        "angular_velocity":["<your-imu-sensor-name-1>"],
+        "linear_velocity": ["<your-gps-sensor-name-1>"],
+        "linear_acceleration": ["<your-accelerometer-sensor-name-1>"]
+      },
+    "depends_on": [] 
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+// { "components": [ { ... },
+{
+    "name": "<your-sensor-name>",
+    "type": "movement_sensor",
+    "model": "merged",
+    "attributes": {
+        "position": ["gps1"],
+        "orientation" : ["vectornav"],
+        "compass_heading" : ["gps1"],
+        "angular_velocity":["vectornav", "mpu6050"],
+        "linear_velocity": ["gps1"],
+        "linear_acceleration": ["adxl345"]
+    }, 
+      "depends_on":[]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Attributes
+
+Configure an array of the `name` of each movement sensor you want to add to your robot as a merged resource in the attributes of the `merged` movement sensor model:
+
+- The name of each attribute represents the `Property` that that particular movement sensor supports, or the type of reading or measurement that it takes.
+- Get the properties supported by each model from its [model configuration documentation](/components/movement-sensor/#configuration), or by calling [`GetProperties()`](/components/movement-sensor/#getproperties) on the sensor.
+- Use this information to determine which attribute to put its `name` inside the array of.
+You can use the same sensor for multiple attributes if it supports multiple properties.
+
+Name | Type | Inclusion | Description
+---- | ---- | --------- | -----------
+`position` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads position. |
```

Comment:
```suggestion
`position` | array | **Dependent on whether Position is  Supported in  GetProperties** | The `name` of the movement sensor you want to merge, if it reads position. |
```
And similar

Reply:
I believe the inclusions here should be sentence case

Reply:
So actually here I am working on putting together "sensor readings" docs pages that talk about what Position and Orientation readings look like (and the other types supported in Properties too) and essentially better instructions on how you should handle the data -- i was thinking of using the "Readings Type" term consistently there so I was gonna link once that's up haha, hard to explain. So might not want to change this because it will make more sense soon to docs users? 

I think that @randhid  suggested change is better right now technically but it does make the table look weird bc it's so long... what do y'all think

Reply:
ah length of table is annoying, "Depends on sensor returning Position"? I'm not super picky

Reply:
I agree the longer ones are too big for the table. I think leave as is for now but sentence case like
```suggestion
`position` | array | **Dependent on readings type supported** | The `name` of the movement sensor you want to merge, if it reads position. |
```